package.json

```diff
@@ -38,7 +38,7 @@
     "@uniswap/default-token-list": "^11.13.0",
     "@uniswap/permit2-sdk": "^1.3.0",
     "@uniswap/router-sdk": "^1.14.0",
-    "@uniswap/sdk-core": "^5.9.0",
+    "@uniswap/sdk-core": "github:alex-stone/uniswap-sdks#alex-stone/add-base-sepolia",
```

Comment:
Looks like the submodule installation doesn't work nicely with the git / github syntax.

Will try to figure that out / hopefully land that PR!